themes/bootstrap3/templates/RecordTab/collectionlist.phtml

```diff
@@ -59,6 +59,16 @@
         <?php endif; ?>
       </div>
     </div>
+    <?php if (!empty($params->getFilterList(false))): ?>
+      <?=$this->render('search/filters.phtml',
+        [
+          'params' => $params,
+          'urlQuery' => $results->getUrlQuery(),
+          'isCollectionTab' => true
```

Comment:
Is there perhaps a more generic variable name we could use? That is, can we describe the effect of this variable rather than explicitly linking the behavior to the collection tab? Seems like it might be more clear and more reusable that way.

Reply:
I renamed it to disableFilterSummary, but I'm not super-happy with that either. Any suggestions?

Reply:
What exactly is the purpose of this variable? I'm not sure I understand what we're trying to hide or why. Are we unable to implement the same behavior in the tab as in the main search for technical reasons, or is this a UI decision?

Reply:
It's both. It doesn't currently work, and it doesn't make much sense since the text field is also a filter.

Reply:
Is `disableRetainFilterControls` more accurate?

Reply:
Yes, thanks, now renamed.